### PR TITLE
update proxy certs

### PR DESCRIPTION
**Pull request by jinyunma**
1. remove openshift-e2e-vsphere-proxy workflow, since it has been added under cucushift folder
2. remove e2e-vsphere-proxy job from release jobs, same job will be added to cucushift jobs later.
3. qe-proxy creds changed to plaintext in vault (keep the same style with other creds under openshift-qe collection in Vault), so update step to adapt.

### CODE REVIEW

**Comment by jinyunma** [comment]
ipi-on-vsphere with http proxy job is passed [here](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/24644/rehearse-24644-periodic-ci-openshift-verification-tests-master-ocp-installer-rehearse-installer-rehearse-vsphere-discon/1471688197346955264).

upi-on-vsphere with https proxy failed due to bz [2031049](https://bugzilla.redhat.com/show_bug.cgi?id=2031049), but I logined to bootstrap and control nodes, https proxy and additional ca are setting correctly. 

@jianlinliu @gpei PTAL, thanks.

File: ci-operator/step-registry/ipi/conf/vsphere/proxy/https/ipi-conf-vsphere-proxy-https-commands.sh [hunk 1]
```diff
@@ -3,21 +3,17 @@ set -o errexit
...
+proxy_creds=$(< /var/run/vault/vsphere/proxy_creds)
+additional_ca="/var/run/vault/vsphere/client_ca.crt"
 
-proxy_user=$(grep -oP 'user\s*:\s*\K.*' ${PROXY_CREDS_PATH})
-proxy_password=$(grep -oP 'password\s*:\s*\K.*' ${PROXY_CREDS_PATH})
-additional_ca=$(cat ${ADDITIONAL_CA_PATH})
-
-http_proxy_url="http://${proxy_user}:${proxy_password}@172.31.249.80:3128"
-https_proxy_url="https://${proxy_user}:${proxy_password}@172.31.249.80:3130"
+http_proxy_url="http://${proxy_creds}@172.31.249.80:3128"
```
Comment (jianlinliu) [inline ci-operator/step-registry/ipi/conf/vsphere/proxy/https/ipi-conf-vsphere-proxy-https-commands.sh:9]: How about saving the bastion IP in the vault too.

Comment (jinyunma) [reply ci-operator/step-registry/ipi/conf/vsphere/proxy/https/ipi-conf-vsphere-proxy-https-commands.sh]: sure, saved ip in the vault, and updated the step.
Failed job is still due to bz 2031049.

**Comment by jinyunma** [comment]
/assign @stbenjam
@stbenjam could you please help to approve this PR? thanks.

**Comment by jinyunma** [comment]
@vrutkovs would you mind to help to review and approve this PR? thanks.

**Comment by liangxia** [comment]
I would suggest break the PR into two PRs. The first PR contains the changes that can handled by QE team, and the other PR contains changes that need DPTP to review/approve.

**Pull request merged**